cli/pkg/timescaledb_secrets/create.go

```diff
@@ -118,6 +120,7 @@ func (t *TSDBSecretsInfo) createTimescaleDBCredentials() error {
 		Type: "Opaque",
 	}
 
+	t.DBPassword = superUserPass
 	return t.K8sClient.CreateSecret(sec)
```

Comment:
I am hard against having to create database connection secret when timescaledb helm chart is no longer requiring pre-created credentials secret. That said I am fine with creating a tech-debt item to remove this later.